src/commands/utility/vote-points.js

```diff
@@ -5,28 +5,48 @@ module.exports = {
   data: new SlashCommandBuilder()
     .setName('vote-points')
     .setDescription(
-      'Start a Effort Estimation Points votation for current Thread'
+      'Start an Effort Estimation Points votation for the current Thread'
     )
     .addUserOption((option) =>
       option
         .setName('user')
         .setDescription('User to search for')
         .setRequired(true)
+    )
+    .addStringOption((option) =>
+      option
+        .setName('point-type')
+        .setDescription('Select the type of points to award')
+        .setRequired(true)
+        .addChoices(
+          { name: 'Normal Points', value: 'normal' },
+          { name: 'Boosted Points', value: 'boosted' }
+        )
     ),
   async execute(interaction) {
     const user = interaction.options.getUser('user') || interaction.user;
+    const pointType = interaction.options.getString('point-type');
+
+    const selectedTagId =
+      pointType === 'boosted' ? tagIds.boostedPointTagId : tagIds.addPointTagId;
 
-    await interaction.reply(`<@&${tagIds.taskCompletedTagId}> <@${user.id}>`);
+    await interaction.reply(
+      `<@&${tagIds.taskCompletedTagId}> <@${user.id}> (Awarding: ${pointType.toUpperCase()} Points with Tag <@&${selectedTagId}>)`
+    );
```

Comment:
Maybe we can remove this interaction?, I thought it was only for debugging, but not sure if it is needed 😄

Reply:
After testing I realized that if it is necessary, if that line is deleted the code is damaged

Reply:
You're right, we can leave the message, just remove the last part

```suggestion
    await interaction.reply(
      `<@&${tagIds.taskCompletedTagId}> <@${user.id}>`
    );
```